internal/cli/kraft/cloud/deploy/deploy.go

```diff
@@ -92,6 +92,24 @@ func NewCmd() *cobra.Command {
 		panic(err)
 	}
 
+	cmd.Flags().Var(
+		cmdfactory.NewEnumFlag[create.RolloutStrategy](
+			append(create.RolloutStrategies(), create.StrategyPrompt),
+			create.StrategyPrompt,
+		),
+		"rollout",
+		"Set the rollout strategy for an instance which has been previously run in the provided service group.",
+	)
+
+	cmd.Flags().Var(
+		cmdfactory.NewEnumFlag[create.RolloutQualifier](
+			create.RolloutQualifiers(),
+			create.RolloutQualifierImageName,
+		),
+		"rollout-qualifier",
+		"Set the rollout qualifier used to determine which instances should be affected by the strategy in the supplied service group.",
+	)
```

Comment:
Why here and not up there? :D

Reply:
What do you mean?

Reply:
Ah nvm, enum flags can't be easily declared with the rest, okay

![image](https://github.com/unikraft/kraftkit/assets/46280801/71fa6eaf-5599-4e6a-b4b9-0e765c0569c5)